src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    private static final int MAX_VALUE_FOR_FIGURES = 26;
+    private static final int MAX_VALUE_FOR_CIRCLE_RADIUS = 10;
+    private Random random = new Random();
+
```

Comment:
This class "FigureSupplier" should not extends ColorSupplier
need to create a class variable
```suggestion
public class FigureSupplier {
    private static final int MAX_VALUE_FOR_FIGURES = 26;
    private static final int MAX_VALUE_FOR_CIRCLE_RADIUS = 10;
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

```